content/en/docs/tutorials/cluster-management/checking-cluster-registration.md

```diff
@@ -34,9 +34,7 @@ status:
   token: REDACTED
 ```
 
-After `ClusterRegistrationRequest` is approved, the status will be updated with corresponding credentials that can be
-used to access parent cluster if needed. Those credentials have been set with scoped RBAC rules, see blow two rules for
-details.
+`ClusterRegistrationRequest` 获准后，用于访问父集群的相应凭据的状态将被更新。这些凭据使用集群或项目空间 RBAC 规则进行设置，有关详细信息，请参阅以下两条规则。
```

Comment:
Still obscure.

Reply:
Thanks, already update, can you have a review?